src/renderer/components/+extensions/extensions.tsx

```diff
@@ -356,7 +356,7 @@ export async function attemptInstallByInfo({ name, version, requireConfirmation
   return attemptInstall({ fileName, dataP }, disposer);
 }
 
-async function attemptInstall(request: InstallRequest, d?: ExtendableDisposer): Promise<void> {
```

Comment:
What are these changes for?

Reply:
Typescript complains about this function when `Notification.error` return function.

Reply:
Ah I think it would be better to change line `373` below to be `return void Notifications.error(...`

Reply:
Updated